api/db/batch.go

```diff
@@ -57,24 +56,12 @@ func (b *Batch) Add(request *Request) (error) {
 		allReq.Type = utils.Create
 	}
 	if req, ok := req.(*Update); ok {
-		findJSON, err := json.Marshal(req.getFind())
-		if err != nil {
-			return err
-		}
-		allReq.Find = findJSON
-		updateJSON, err := json.Marshal(req.getUpdate())
-		if err != nil {
-			return err
-		}
-		allReq.Update = updateJSON
+		allReq.Find = map[string]interface{}{"find": req.getFind()}
+		allReq.Update = map[string]interface{}{"update": req.getUpdate()}
 		allReq.Type = utils.Update
 	}
 	if req, ok := req.(*Delete); ok {
-        findJSON, err := json.Marshal(req.getFind())
-		if err != nil {
-			return err
-		}
-		allReq.Find = findJSON
+		allReq.Find = map[string]interface{}{"find": req.getFind()}
```

Comment:
```suggestion
		allReq.Find = req.getFind()
```

Reply:
i have changed the AllRequest struct ,in that i changed Find & Delete field from map[string]interface{} to interface only